### ISSUE TITLE
nyc output indicates 40% for '% Funcs' column.  But I'd expect it to be 100%. Why the discrepancy?

### ISSUE THREAD

**Issue by cagross** [istanbuljs/nyc#1390]
## Link to bug demonstration repository

https://github.com/cagross/test-nyc

## Expected Behavior

nyc coverage report indicates 100% in the `% Funcs` column.  But the figure in my report is 40%.  I don't understand the discrepancy.

## Observed Behavior

nyc coverage report indicates 40% in the `% Funcs` column.  

Here are the steps I took to reproduce.

* In my project I have a file `lib/ow/css.js` with three simple functions defined: `lib/ow/css.js`.  In a separate file (`test/lib/ow/css.test.js`) I have written unit tests which call all three functions. 

* I instrumented the file with the command:
`npx nyc instrument lib/ow/css.js > cagtest/output.js`

* I generated the code coverage report with this command:
`npx nyc --reporter=text --report-dir=./nyc_output node cagtest/output.js`

**Of Note**

* I noticed that in the report, the line in-question seems to be reporting data for the file `cagtest/css.js`, which isn't a file that exists. It seems like this filename should be `cagtest/output.js`.  Not sure if that's expected, and/or related to my issue.

* My test runner is [Tape][6].

  [6]: https://github.com/substack/tape

## Environment Information

```
# paste the output here

$ npx envinfo@latest --preset nyc

npx: installed 1 in 1.77s

  System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i7-2600 CPU @ 3.40GHz
    Memory: 2.50 GB / 7.88 GB
  Binaries:
    Node: 14.15.1 - C:\Program Files\nodejs\node.EXE
    npm: 6.14.8 - C:\Program Files\nodejs\npm.CMD
  npmPackages:
    nyc: ^15.1.0 => 15.1.0
```

**Comment by coreyfarrell** [comment]
You're not actually running any tests.  Also from the repo you've shared you should not be using `nyc instrument`.  You should be running `nyc tape test/lib/ow/css.js`.  Don't preinstrument, this is something which supports a different use case of istanbul and is likely the source of your incorrect filename path reported.

**Comment by cagross** [comment]
OK thanks very much for the info.  

> You should be running `nyc tape test/lib/ow/css.js`

I ran `npx nyc tape test/lib/ow/css.js` and the output is a table with a single row (All files), and each column has a 0 in it ([screenshot](https://i.imgur.com/QMGFwIG.jpeg)).  What am I misunderstanding?  Is it still the case that this command isn't running any tests?  If so, how do I resolve that?

**Comment by coreyfarrell** [comment]
I'm not sure what the issue is, I just cloned your repo, ran `npm install && npx nyc tape test/lib/ow/css.js`, it showed me 100% coverage for your script.

Note I've seen issues with running nyc under npx on windows in the past (I don't run windows).  You might try setting your test script to `nyc tape test/lib/ow/css.js` then running `npm test`.

**Comment by cagross** [comment]
Thanks very much for the help so far.

> I'm not sure what the issue is, I just cloned your repo, ran npm install && npx nyc tape test/lib/ow/css.js, it showed me 100% coverage for your script.

OK interesting.  That's promising then.

> Note I've seen issues with running nyc under npx on windows in the past (I don't run windows). You might try setting your test script to nyc tape test/lib/ow/css.js then running npm test.

I have this a shot.  In `package.json` I set:

`"mynycscript": "nyc tape test/lib/ow/css.js"`

From the command line I then ran:  `npm run mynycscript`  But the same blank table was output.

I also have the following to report:

* As a test, from the command line, I ran `node node_modules/nyc/bin/nyc` The output was the usual list of nyc help commands.
* From the command line I then ran `node node_modules/nyc/bin/nyc tape test/lib/ow/css.js`.  But this failed with:

```
'tape' is not recognized as an internal or external command,
operable program or batch file.
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error: spawn tape ENOENT
    at notFoundError (C:\Users\Snarl\development-snarl\test-coverage\node_modules\cross-spawn\lib\enoent.js:6:26)
    at verifyENOENT (C:\Users\Snarl\development-snarl\test-coverage\node_modules\cross-spawn\lib\enoent.js:40:16)
    at ChildProcess.cp.emit (C:\Users\Snarl\development-snarl\test-coverage\node_modules\cross-spawn\lib\enoent.js:27:25)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
Emitted 'error' event on ChildProcess instance at:
    at ChildProcess.cp.emit (C:\Users\Snarl\development-snarl\test-coverage\node_modules\cross-spawn\lib\enoent.js:30:37)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12) {
  code: 'ENOENT',
  errno: 'ENOENT',
  syscall: 'spawn tape',
  path: 'tape',
  spawnargs: [ 'test/lib/ow/css.js' ]
}
```

Thoughts?

**Comment by cagross** [comment]
Also, I have one more thing I can report.  Back in the original repo (as opposed to this test repo you and I are using), it seemed that nyc had been uninstalled, i.e. running `npm list nyc` resulted in:

``-- (empty)`

BUT, before I realized this, I ran `npx nyc tape test/lib/ow/css.js`, and there was no catastrophic failure alerting me that nyc was not installed.  Rather, the exact same blank table printed to the console.  The same thing occurred when I created a `"mynycscript": "nyc tape test/lib/ow/css.js"`.

I then ran `npm install`, confirmed nyc was properly installed, and tried the same commands.  But there was no change--the same blank tables were printed to the console.
